expression/builtin_string_vec_test.go

```diff
@@ -87,10 +87,21 @@ var vecBuiltinStringCases = map[string][]vecExprBenchCase{
 		{retEvalType: types.ETString, childrenTypes: []types.EvalType{types.ETString}, geners: []dataGenerator{&randSpaceStrGener{10, 100}}},
 		{retEvalType: types.ETString, childrenTypes: []types.EvalType{types.ETString, types.ETString}, geners: []dataGenerator{&randLenStrGener{10, 20}, &randLenStrGener{5, 25}}},
 	},
-	ast.LTrim:     {},
-	ast.RTrim:     {},
-	ast.Lpad:      {},
-	ast.Rpad:      {},
+	ast.LTrim: {},
+	ast.RTrim: {},
+	ast.Lpad:  {},
+	ast.Rpad: {
+		{
+			retEvalType:   types.ETString,
+			childrenTypes: []types.EvalType{types.ETString, types.ETInt, types.ETString},
+			geners:        []dataGenerator{&randLenStrGener{0, 20}, &rangeInt64Gener{0, 20}, &randLenStrGener{0, 10}},
```

Comment:
How about adding some cases to test this check this condition `uint64(targetLength)*uint64(mysql.MaxBytesOfCharacter) > b.maxAllowedPacket`?

Reply:
Ok